backend/src/auth/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const jwtConstants = {
+  secretKey: 'maTe85_2022&2=foLl0Vvup$chOl4rsh!p',
+  expirationTime: '60s'
```

Comment:
Pelo q entendo vamos retornar um JWT quando o usuário fazer o login, esse JWT será colocado no header do usuário quando ele fzr loguin, assim todas as requisições q ele fizer a partir daí conseguimos pegar esse JWT e saber quem é o usuário que está logado fzd a requisição. Se o expirationTime for somente 60 segundos significa q dps de 60 segundos ele tem q fazer loguin novamente. Eu colocaria aqui no minímo 1 dia.

Reply:
Concordo! mais de um dia é o ideal
E por ser um dado sensível, aproveita e cria uma env para a secretKey, dai vc puxa da env e põe ai